shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/apache/ApacheClientTest.groovy

```diff
@@ -6,26 +6,6 @@ import spock.lang.Specification
 
 class ApacheClientTest extends Specification {
 
-    def "convertMapToHeader works"() {
-        given:
-        def headerMap = [
-            "key": "value",
-            "name": "dogCow",
-            "first": "last"
-        ]
-        def bhArr = [
```

Comment:
Would it be worth commenting these tests and having an item to uncomment them in the devex/opex backlog in the future when we get bumped to a bug free version?

Reply:
@jcrichlake I think that's a good idea

Reply:
Since the `org.apache.hc.core5.http.Header` is not supported in the 5.3.x version, we will not be utilizing the `convertMapToHeader(Map<String, String> headerMap)` helper function. This unit test was designed to test the  functionality of the helper function. Once the helper function gets deleted, there is no need for this test case.

Reply:
Gotcha, thanks for clarifying